manifests/client.pp

```diff
@@ -9,10 +9,10 @@
 #   is repository dependent.
 #
 class mongodb::client (
-  $ensure       = $mongodb::params::package_ensure_client,
-  $package_name = $mongodb::params::client_package_name,
+  Variant[Boolean, String] $ensure = $mongodb::params::package_ensure_client,
```

Comment:
Boolean?

Reply:
Some use ```false```. Do we have any convention in voxpupuli?

Reply:
I based this on params - `package_ensure_client` has both `$version` and `true`. This module uses some weird patterns. I think we may want to try and fix some of the weird design decisions (though some of them would require major structural work), but for this pass, I'm just trying to keep things working.

Reply:
Boolean looks really wrong here. But as a first step we should keep the current behavior to make a migration to datatypes easier.

Reply:
Here's where it's used  https://github.com/voxpupuli/puppet-mongodb/blob/master/manifests/params.pp#L45-L61
I think this is also causing some quirks when installing versions, because of vendor release tags etc.